psydac/api/ast/linalg_kernels.py

```diff
@@ -1,13 +1,14 @@
+from pyccel.decorators import template
 #========================================================================================================
-def transpose_1d( M:'float[:,:]', Mt:'float[:,:]', n1:"int64", nc1:"int64", gp1:"int64", p1:"int64",
+@template(name='T',types=['float[:,:]','complex[:,:]'])
```

Comment:
Please add an empty space after the commas. This applies to all the `template` instructions in this file.

Reply:
[86b06e9](https://github.com/pyccel/psydac/pull/273/commits/86b06e97fda7a028f2d6ec3f23b6c81d44eaf696)